simulation/adx_gyro_accelerometer/src/main/native/cpp/ADXRS450_SpiGyroWrapperData.cpp

```diff
@@ -7,26 +7,51 @@
 
 #include "ADXRS450_SpiGyroWrapperData.h"
 
+#include <cmath>
 #include <cstring>
 
 #include "MockData/NotifyCallbackHelpers.h"
 #include "MockData/SPIData.h"
 
+#ifdef _WIN32
+#include "Winsock2.h"
+#pragma comment(lib, "ws2_32.lib")
+#else
+#include <arpa/inet.h>
+#endif
+
 using namespace hal;
 
-const double ADXRS450_SpiGyroWrapper::ANGLE_LSB = 1 / 0.0125 / 0.001;
+const double ADXRS450_SpiGyroWrapper::kAngleLsb = 1 / 0.0125 / 0.0005;
+const double ADXRS450_SpiGyroWrapper::kMaxAngleDeltaPerMessage =
+    0.1875;  // Close to saturating available bits, but not totally
```

Comment:
If the comment is too long, you could put it on the previous line instead with an empty line before it. Also, available bits of what?

Reply:
Updated the comment.  It is approximately the maximum number you can fit in 15 bits that are available for data.  Put some fudge factor in there, adding an extra loop or two won't degrade peformance